### DISCUSSION TITLE
Projects Beta Workflow: Code changes requested seems doesn't work

### DISCUSSION THREAD

**Discussion by andreyluchianic** [community/community#19804] [Projects and Issues]
Other workflows seems to work OK only review:changes_requested doesn't trigger 

<img width="1357" alt="Screenshot 2022-07-01 at 11 07 25" src="https://user-images.githubusercontent.com/88837148/176852845-22f654fa-0362-44bb-8355-333c31e75d13.png">
<img width="594" alt="Screenshot 2022-07-01 at 11 08 04" src="https://user-images.githubusercontent.com/88837148/176853233-93163bd2-d95a-49c1-9c29-d85243178e8f.png">
<img width="1028" alt="Screenshot 2022-07-01 at 11 08 23" src="https://user-images.githubusercontent.com/88837148/176853006-0e08a6ec-3380-4cb8-82d0-c39c983ffc4c.png">

**Comment by Fryuni** [comment]
Since the preview stage, only the "Item closed" workflow has worked for us. The others seem to have no effect at all.
